src/OperationProcessor.ts

```diff
@@ -76,576 +18,145 @@ export interface OperationProcessor {
    * Process a DID write (state changing) operation with the guarantee
    * that any future resolve for the same DID sees the effect of the
    * operation.
-   * @returns the hash of the operation.
    */
-  process (operation: Operation): Promise<string>;
+  process (operation: Operation): Promise<void>;
 
   /**
    * Remove all previously processed operations with transactionNumber
-   * greater than the provided parameter value.
+   * greater or equal to the provided parameter value.
```

Comment:
This changes the current contract with the Observer, is it possible to keep it the same? We went over the technical reason why a while back, but we can talk about it again.

Reply:
I simply forgot if we had decided on >= or >. Will switch to >.